stdlib/public/Concurrency/Task+immediate.swift.gyb

```diff
@@ -255,6 +390,7 @@ extension Task where Failure == ${FAILURE_TYPE} {
   @MainActor
   @available(SwiftStdlib 5.9, *)
   @discardableResult
+  @available(*, deprecated, renamed: "immediate")
```

Comment:
The new equivalent of this is `Task.immediate { @MainActor }` or just Task.immediate from a context that's dynamically/statically on the main actor, so it's time to deprecate this SPI